src/Elastic.Apm/DiagnosticListeners/HttpDiagnosticListenerImplBase.cs

```diff
@@ -124,7 +124,7 @@ private void ProcessStartEvent(TRequest request, Uri requestUrl)
 
 			var currentExecutionSegment = _agent.Tracer.CurrentSpan ?? (IExecutionSegment)transaction;
 			var span = currentExecutionSegment.StartSpan(
-				$"{RequestGetMethod(request)} {requestUrl.Host}",
+				$"{RequestGetMethod(request)} {requestUrl.AbsolutePath}",
```

Comment:
I looked into this in more detail, and here is what I can say:

To get to the issue:

> Because we are more concerned about which APIs are called and not which hosts when we analyze the call link.

You can figure this out by clicking on the method span - we show the whole URL which we set later in the cade [here](https://github.com/elastic/apm-agent-dotnet/blob/master/src/Elastic.Apm/DiagnosticListeners/HttpDiagnosticListenerImplBase.cs#L144).

<img width="702" alt="Screen Shot 2019-08-30 at 16 58 43" src="https://user-images.githubusercontent.com/1091853/64031304-68eda780-cb48-11e9-963b-4a350e289188.png">


I know this is 1 click more than your solution, but the thinking behind this is to avoid overloading the UI - overall you definitely have the information you need.

Also, we try to standardize the span names across agents. We have [this](https://github.com/elastic/apm/blob/master/docs/agent-development.md#http-client-spans) document stating that:

> span name should have the format `<method> <host>`.

We also have [this spreadsheet](https://docs.google.com/spreadsheets/d/1SmWeX5AeqUcayrArUauS_CxGgsjwRgMYH4ZY8yQsMhQ/edit#gid=644582948) and as you can see there, all agents follow this format, except the Node.js agent - I’m trying to figure out why.

Now, specifically this line would introduce yet another format, so in this case it’d be `<method> <path>` and currently no other agent generates span name based on this format. I still think whether we should change this, but in case we do, I’d suggest to at least do what the Node.js agent does and also include the host, so the span name should be something like `<method> <host><path>`. I’m talking to other agent devs to figure out what we want to do here - happy to hear opinions on this.